src/storage/v2/inmemory/unique_constraints.cpp

```diff
@@ -273,6 +274,44 @@ void InMemoryUniqueConstraints::UpdateBeforeCommit(const Vertex *vertex, const T
   }
 }
 
+void InMemoryUniqueConstraints::UpdateBeforeCommit(const Vertex *vertex, std::unordered_set<LabelId> &added_labels,
+                                                   std::unordered_set<PropertyId> &added_properties,
+                                                   const Transaction &tx) {
+  std::vector<LabelId> added_labels_vec{};
+  added_labels_vec.reserve(added_labels.size());
+  added_labels_vec.insert(added_labels_vec.end(), added_labels.begin(), added_labels.end());
+
+  auto labels = !added_properties.empty() ? vertex->labels : added_labels_vec;
+  for (const auto &label : labels) {
+    if (!constraints_by_label_.contains(label)) {
+      continue;
+    }
+
+    for (auto &[props, storage] : constraints_by_label_.at(label)) {
+      if (added_labels.empty()) {
```

Comment:
I'm not really following what's going on here.
Sorry :D

Reply:
oops. i wanted to delete this, I'm terribly sorry.